src/app/services/navigation/navigation.service.ts

```diff
@@ -23,6 +23,8 @@ export class NavigationService {
   readonly hasEnclosure$ = this.store$.select(selectHasEnclosureSupport);
   readonly hasVms$ = new BehaviorSubject(false);
   readonly hasApps$ = new BehaviorSubject(false);
+  readonly hasFibreChannel$ = new BehaviorSubject(false);
+  readonly hasDedup$ = new BehaviorSubject(false);
```

Comment:
I think it's weird to keep adding new fields here. They are not used in this service and are not really about Navigation.
It's probably time to extract all of this into a separate service.